test/xml2json.test.js

```diff
@@ -73,6 +76,10 @@ describe('Examples', function () {
     it('documentation-v2', function () {
         assert.deepStrictEqual(csdl.xml2json(example9), result9, 'CSDL JSON');
     })
+
+    it('function', function () {
+        assert.deepStrictEqual(csdl.xml2json(example10), result10, 'CSDL JSON');

```

Comment:
```suggestion
        assert.deepStrictEqual(csdl.xml2json(example10, true), result10, 'CSDL JSON');
```

Should request line numbers, and desired output should contain them.

You could just extend the existing test case for line numbers by adding parameters and return types, see https://github.com/oasis-tcs/odata-csdl-schemas/blob/0a4dcac286d032452849ab8d0778625167313e80/test/xml2json.test.js#L118